strongbox-client/pom.xml

```diff
@@ -94,7 +94,11 @@
             <groupId>org.glassfish.jersey.connectors</groupId>
             <artifactId>jersey-apache-connector</artifactId>
         </dependency>
-
+        <dependency>
+            <groupId>org.glassfish.jersey.inject</groupId>
+            <artifactId>jersey-hk2</artifactId>
+            <version>2.26</version>
```

Comment:
Could you please, move this version definition to the parent and leave it versionless here? Thanks! :)

Reply:
Fixed.